packages/core/src/utils/slack.ts

```diff
@@ -52,26 +52,49 @@ export function userIdBlock(userId: string): SectionBlock {
   };
 }
 
-export function userButtonsBlock(userId: string): ActionsBlock {
-  return {
-    type: "actions",
-    elements: [
-      {
-        type: "button",
-        text: {
-          type: "plain_text",
-          text: "Posthog User",
+export function actionsBlock({
```

Comment:
Initially i just wanted to add another block ... but that was failing silently and it turns out you're only allowed one actions block per message